evm/src/keccak_sponge/keccak_sponge_stark.rs

```diff
@@ -479,7 +473,13 @@ impl<F: RichField + Extendable<D>, const D: usize> Stark<F, D> for KeccakSpongeS
         );
         yield_constr.constraint(builder, constraint);
 
-        let is_final_block = local_values.is_final_block;
+        let mut is_final_block = builder.add_extension(
```

Comment:
I believe starting with an accumulator of zero should give the same performance; addition with zero should be skipped by `arithmetic_extension_special_cases`. There's also an `add_many_extension` method we could use for brevity.

Reply:
I opted for `add_many_extension`, thanks for the tip.